layersvt/api_dump_layer.md

```diff
@@ -105,6 +105,7 @@ Detailed Output | `VK_APIDUMP_DETAILED` | `lunarg_api_dump.detailed` | true | Ge
 No Addresses/Handles | `VK_APIDUMP_NO_ADDR` | `lunarg_api_dump.no_addr` | false | Generate output without addresses or handles (which can vary run to run. Instead use the placeholder value "address".
 Flush After Every Command | `VK_APIDUMP_FLUSH` | `lunarg_api_dump.flush` | true | Flush after every API command's output
 Output format | `VK_APIDUMP_OUTPUT_FORMAT` | `lunarg_api_dump.output_format` | `text` | Output the API Dump information as a text file (`text`), an HTML-formated file (`html`), or a json file (`json`).
+Show Timestamps | `VK_APIDUMP_TIMESTAMP` | `lunarg_api_dump.show_timestamp` | true | Show the timestamp of function calls since start in microseconds
```

Comment:
start of what? Might be better to describe it as "Show a timestamp for function calls. Timestamps are microseconds relative to the first Vulkan function call."

Reply:
It starts the clock whenever API Dump is initialized. I'll amend it to reflect that.

Reply:
Amended.